grpc/src/main/java/io/stargate/grpc/service/ExceptionHandler.java

```diff
@@ -72,11 +74,37 @@ public void handleException(Throwable throwable) {
     }
   }
 
+  /**
+   * @param status
+   * @param throwable
+   * @return generic error message for gRPC exceptions using gRPC Status and Exception thrown. If
+   *     the status is null and throwable is a {@link StatusException} or {@link
+   *     StatusRuntimeException} it extracts its status. If the status is null and throwable is
+   *     another instance type, it will have the status code {@code Status.UNKNOWN}.
+   */
+  protected String getErrorMessage(
+      @Nullable Status status, @Nonnull Throwable throwable, boolean logStackTrace) {
```

Comment:
remove logStackTrace and add exception as the parameter to the log, then the stacktrace will be logged by the logger.